.changeset/pre.json

```diff
@@ -0,0 +1,22 @@
+{
+  "mode": "pre",
+  "tag": "premajor",
```

Comment:
This should be `alpha` or `alpha-v5` I think
```suggestion
  "tag": "premajor",
```

Reply:
done!